internal/oci/uvm.go

```diff
@@ -395,5 +395,11 @@ func UpdateSpecFromOptions(s specs.Spec, opts *runhcsopts.Options) specs.Spec {
 		s.Annotations[annotations.NetworkConfigProxy] = opts.NCProxyAddr
 	}
 
+	for key, value := range opts.DefaultContainerAnnotations {
```

Comment:
It's weird this code lives in `uvm.go`, given it is not at all specific to UVMs, or hv-isolation. Perhaps we can address that in a future change.